components/events/AccordionSection.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <section
+    class="accordion__section"
+  >
+    <h2 class="copy__title">
+      {{ document.title }}
+    </h2>
+    <cv-accordion :align="`end`">
+      <nuxt-content :document="document" />
+    </cv-accordion>
+  </section>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component, Prop, Watch } from 'vue-property-decorator'
+import { IContentDocument } from '@nuxt/content/types/content'
+
+@Component
+export default class AccordionSection extends Vue {
+  @Prop({ type: String }) title: string | undefined
```

Comment:
If this is also expecting "undefined", please expand the information in `@Prop` by adding the `required` and `default` properties.